src/features/areas/components/PublicAreaMap.tsx

```diff
@@ -384,6 +332,9 @@ const PublicAreaMap: FC<PublicAreaMapProps> = ({ area }) => {
             setAnchorEl(null);
             setSelectedPlace(null);
           }}
+          onEdit={() => {
+            setDialogStep('edit');
+          }}
```

Comment:
I find this to be a little bit of a weird separation of concerns. The edit mode is only ever accessed from within the modal, but still it requires the outside of the modal (`PublicAreaMap`) to know about and maintain it's state. It's not a big deal and because we are trying to iterate quickly I don't think it's worth changing at this point, but I think it's something to think about and learn from.